themes/src/main/resources/theme/base/login/webauthn-register.ftl

```diff
@@ -128,13 +128,12 @@
                     });
             }
 
-            function getPubKeyCredParams(signatureAlgorithms) {
+            function getPubKeyCredParams(signatureAlgorithmsList) {
                 let pubKeyCredParams = [];
-                if (signatureAlgorithms === "") {
+                if (signatureAlgorithmsList === []) {
```

Comment:
This will always be false. To test for an empty array in JS, test for its length being zero.

```suggestion
                if (signatureAlgorithmsList.length === 0) {
```

Fwiw, instead of branching the return here, I think I'd rather push `-7` to the `signatureAlgorithmsList` then let the rest of the algorithm process it like for an non-empty initial list.